website/src/views/DashboardHome/DashboardHome.jsx

```diff
@@ -6,7 +6,13 @@ import './DashboardHome.css';
 
 class DashboardHome extends PureComponent {
   static propTypes = {
-    userData: PropTypes.shape({}).isRequired,
+    userData: PropTypes.shape({}),
+    viewEvent: PropTypes.func,
+  }
+
+  static defaultProps = {
+    userData: null,
```

Comment:
Would there be a situation where the dashboardhome would be displayed with no userdata available?

Reply:
Well i guess if the user deleted their cache and tried to access the page offline

Reply:
props propagation delay causes err. this is to combat that